### PR TITLE
Added more Maya materials

### PR DESCRIPTION
**Pull request by Sanchikuuus**
Added Maya materials to list and add setting texture properties for getting lost textures to mesh

### CODE REVIEW

File: include/assimp/material.h [hunk 1]
```diff
@@ -292,6 +292,11 @@ enum aiTextureType {
     aiTextureType_DIFFUSE_ROUGHNESS = 16,
     aiTextureType_AMBIENT_OCCLUSION = 17,
 
+    aiTextureType_MAYA_BASE = 22,
```
Comment (turol) [inline include/assimp/material.h:295]: Why not use the existing types?

Comment (Sanchikuuus) [reply include/assimp/material.h:295]: Existing types is not including support of Specular/Gloss textures.
Screenshot 2023-05-22 at 14 02 01

**Review by kimkulling — APPROVED** [review]
Looks fine.

**Comment by kimkulling** [comment]
You need to adapt the unittests as well.

**Review by kimkulling — CHANGES_REQUESTED** [review]
Unittests are broken.

**Comment by kimkulling** [comment]
Merged, thanks a lot for your contribution.

**Pull request merged**